### PR TITLE
✨ apply event of original trip when joining

### PR DESCRIPTION
**Pull request by thilobillerbeck**
This PR improves the join functionality by applying the event of the original trip when joining.

### CODE REVIEW

File: resources/js/appControls.js [hunk 1]
```diff
@@ -16,6 +16,12 @@ $(document).on("click", ".join", function (event) {
         modal.find("#input-arrival").val(source.trwlArrival);
         modal.find("#input-start").val(source.trwlStart);
         modal.find("#input-departure").val(source.trwlDeparture);
+        // case for small number of events
+        modal.find("#event_check").each(function () {
+            $(this).prop("checked", $(this).val() == source.trwlEventId);
```
Comment (MrKrisKrisu) [inline resources/js/appControls.js]: Can this be applied here?
`Expected '===' and instead saw '=='.`

**Review by MrKrisKrisu — CHANGES_REQUESTED** [review]

**Review by MrKrisKrisu — APPROVED** [review]
<3

**Comment by MrKrisKrisu** [comment]
> **warning**
> This branch has conflicts that must be resolved

**Comment by thilobillerbeck** [comment]
The conflicts seem to be resolved, just tested it locally

**Pull request merged**